river/render.zig

```diff
@@ -143,8 +143,10 @@ pub fn renderOutput(output: *Output) void {
     }
 
     // TODO: handle failure
-    output.wlr_output.commit() catch
-        log.err("output commit failed for {}", .{output.wlr_output.name});
+    output.wlr_output.commit() catch {
+        log.err("output commit failed for {}, retrying", .{output.wlr_output.name});
```

Comment:
Lets fix this to stop at the null terminator instead of printing the full array since we're already touching this line.
```suggestion
        log.err("output commit failed for {}, retrying", .{std.mem.spanZ(&output.wlr_output.name)});
```